docs/pages/architecture/deployment-model.md

```diff
@@ -6,20 +6,26 @@ status: 'published'
 
 # Deployment Model
 
-The primary deployment model is to run Fennel in a fully managed mode inside 
-your VPC as a single tenant system -- as a result, the data never leaves your
-cloud and is fully covered by your existing infosec policies. Over time, a hosted
-multi-tenancy system will also be made available.
+The primary deployment model is to run Fennel in a fully-managed mode inside
+your cloud account as a single-tenant system. As a result, the data never leaves your
+cloud.
 
+**Deployment on AWS**
 
-**How does deployment work inside your VPC?**
-
-You create a sub-organization in your cloud and give Fennel some privileges to it.
-Fennel can then bring up machines/services and scale things up and down as
-needed to handle the traffic. Fennel cluster inside your VPC talks to Fennel control
-plane only to exchange telemetry information and control instructions (e.g. control plane
-may choose to downscale the dataplane when traffic is lower, say at night). Either way,
-your code and data never leave your cloud premise.
-
-Since Fennel is limited to a sub-organization, it has no visibility / access into the
-rest of your cloud.
+1. Create a new account in
+   your [AWS Organization](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_introduction.html).
+2. Create a role in the new account and assign it admin privilege by attaching
+   the
+   AWS-managed [AdministratorAccess policy](https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AdministratorAccess.html)
+   to it
+3. Allow Fennel to assume this role by adding `arn:aws:iam::030813887342:root`
+   as a trusted entity for the role created in the previous step
+4. Provide the AWS account id and role name to Fennel. Optionally, also provide a `/16` VPC CIDR
+   block that does not overlap with any VPC on your end that we might want to peer with. Fennel will create a VPC in
+   your account with this CIDR block.
+5. Fennel deploys its Feature Engineering Platform in this account and manages
+   it from there. To deploy and perform ongoing maintenance and upgrades,
+   Fennel peers the deployed VPC with a VPC in our account (a.k.a. the control plane)
```

Comment:
Remove 'our' - we don't use 1st person while referring to Fennel in the documentation.